kanidmd/src/lib/entry.rs

```diff
@@ -1293,6 +1293,9 @@ impl Entry<EntrySealed, EntryCommitted> {
         }
     }
 
+    // ! TRACING INTEGRATED
+    // Why is this returning a `Result<Self, ()>` when we could just do `Option<Self>`
```

Comment:
It doesn't XD

I'm going to fix it this week, I have 4 clippy lints left that need fixing and this is one.